HEAD doesnt build

problem in src/modules/rlm_mschap/auth_wbclient.c

line 109

authparams.parameter_control |= WBC_MSV1_0_ALLOW_MSVCHAPV2;


CC src/modules/rlm_mschap/auth_wbclient.c
src/modules/rlm_mschap/auth_wbclient.c: In function ‘do_auth_wbclient’:
src/modules/rlm_mschap/auth_wbclient.c:109:34: error: ‘WBC_MSV1_0_ALLOW_MSVCHAPV2’ undeclared (first use in this function)
  authparams.parameter_control |= WBC_MSV1_0_ALLOW_MSVCHAPV2;
                                  ^
src/modules/rlm_mschap/auth_wbclient.c:109:34: note: each undeclared identifier is reported only once for each function it appears in
make: *** [build/objs/src/modules/rlm_mschap/auth_wbclient.lo] Error 1


samba 4.3.1 (also tested with samba 4.3.4)

--- a/src/modules/rlm_mschap/rlm_mschap.c
+++ b/src/modules/rlm_mschap/rlm_mschap.c
@@ -38,6 +38,11 @@ RCSID("$Id$")
 #include "smbdes.h"
 #include "auth_wbclient.h"
 
+/* Samba does not export this constant yet */
+#ifndef WBC_MSV1_0_ALLOW_MSVCHAPV2
+#define WBC_MSV1_0_ALLOW_MSVCHAPV2 0x00010000
+#endif
+
 #ifdef HAVE_OPENSSL_CRYPTO_H
 USES_APPLE_DEPRECATED_API	/* OpenSSL API has been deprecated by Apple */
 #  include	<openssl/rc4.h>

